packages/govuk-frontend/tasks/scripts.mjs

```diff
@@ -12,10 +12,23 @@ import gulp from 'gulp'
 export const compile = (options) =>
   gulp.series(
     /**
-     * Compile GOV.UK Frontend JavaScript for all entry points
+     * Compile GOV.UK Frontend JavaScript for component entry points
      */
-    task.name("compile:js 'modules'", () =>
-      scripts.compile('**/{all,components/*/!(*.test)}.mjs', {
+    task.name("compile:js 'components'", () =>
```

Comment:
We've split the old task into two builds one after the other:

1. Components build first as `compile:js 'components'`
2. Main entry point second as `compile:js 'entry'`